docs/en/ingest-management/fleet/api-generated/template/index.mustache

```diff
@@ -0,0 +1,170 @@
+////
+This content is generated from the open API specification.
+Any modifications made to this file will be overwritten.
+////
+
+++++
+<div class="openapi">
+  <h2>Access</h2>
+  {{#hasAuthMethods}}
+    <ol>
+    {{#authMethods}}
+      <li>{{#isBasic}}HTTP Basic Authentication{{/isBasic}}{{#isOAuth}}OAuth AuthorizationUrl:{{authorizationUrl}}TokenUrl:{{tokenUrl}}{{/isOAuth}}{{#isApiKey}}APIKey KeyParamName:{{keyParamName}} KeyInQuery:{{isKeyInQuery}} KeyInHeader:{{isKeyInHeader}}{{/isApiKey}}</li>
+    {{/authMethods}}
+    </ol>
+  {{/hasAuthMethods}}
```

Comment:
The "access" block seems to render a bit strangely, but maybe this is intentional? Seems like this might just be debugging info for now 🙂 

![image](https://user-images.githubusercontent.com/6766512/221650818-57adf1e6-304c-4853-b183-d137f00d8a0d.png)

Reply:
That does look strange! It turns out that we can edit the `index.mustache` file directly (the warning at the top of the file says that it's generated, but my changes aren't overwritten when I regenerate the API docs). So, I just removed this section, or if we want it seems we can add anything we want here. I've pushed a new commit so the preview should refresh soon.